pkg/wakaama/Makefile

```diff
@@ -12,6 +12,9 @@ patch: git-download
 	mkdir -p "$(PKG_BUILDDIR)/riotbuild"
 	cp $(PKG_BUILDDIR)/core/*.c $(PKG_BUILDDIR)/core/*.h $(PKG_BUILDDIR)/riotbuild
 	cp $(PKG_BUILDDIR)/core/er-coap-13/*.c $(PKG_BUILDDIR)/core/er-coap-13/*.h $(PKG_BUILDDIR)/riotbuild
+	cp $(PKG_BUILDDIR)/examples/client/object_server.c $(PKG_BUILDDIR)/riotbuild
```

Comment:
Sadly this approach that you want to use is not valid. 

The package can not be modified for each application or example.

Reply:
The object implementations taken from the example client should not need to be modified for each application, as they implement a standardized data model but nothing platform or application specific. They each expose functions to create/destroy object instances with customized data.

That is also why I did not take `object_device.c`, as that actually is an object that needs to be modified for each device/application.